app/models/cart_item.rb

```diff
@@ -0,0 +1,24 @@
+class CartItem
+  attr_reader :id, :quantity
+
+  delegate :title, :price, :image, to: :donation
```

Comment:
👍 Great work. I'm really happy you go this together. I hope you see how this pattern is really beneficial to clean refactored code.